packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -91,15 +91,18 @@
     color: var(--terra-notification-dialog-title-color, #000);
     flex: 0 0 auto;
     flex-wrap: wrap;
-    font-size: var(--terra-notification-dialog-title-font-size, 1rem);
+    font-size: var(--terra-notification-dialog-title-font-size, 1.07142rem);
     font-weight: var(--terra-notification-dialog-title-font-weight, bold);
+    /* stylelint-disable terra/custom-property-name*/
```

Comment:
You should be able to remove the stylelint-disable now

Reply:
Updated here 9aa2f48d020806cf0e108693103b1253bb02064a